build/Headers/Headers.Shared/Internal.Shared.vcxitems

```diff
@@ -16,4 +16,9 @@
   <ItemGroup>
     <ProjectCapability Include="SourceItemsFromImports" />
   </ItemGroup>
+  <ItemGroup>
+    <ClInclude Include="$(MSBuildThisFileDirectory)..\..\..\Frameworks\UIKit\WYPopoverController.h" />
```

Comment:
I had the headers in the wrong directory. Now that I've moved them to Frameworks/include it was possible to remove those lines from the vcxitems.